packages/cxl-lumo-styles/scss/themes/vaadin-horizontal-layout.scss

```diff
@@ -5,3 +5,17 @@
     color: var(--lumo-primary-color);
   }
 }
+
+:host([theme~="cxl-marketing-hero-details"]) {
+  font-size: var(--lumo-font-size-s);
+
+  ::slotted(span) {
+    display: inline-flex;
+    align-items: center; /* In Firefox, for some weird reason, things look more center *without* this property. */
```

Comment:
My comment sort of expected you to confirm, and probably remove it. What do you see in your visual testing?

Reply:
> What do you see in your visual testing?

I didn't test

Reply:
You don't have to, I did it for you.

More eyes with Chrome is enough to confirm.